includes/admin/class-wcmp-admin.php

```diff
@@ -691,6 +734,24 @@ private function renderStatus(array $shipment): void
     {
         echo $shipment["status"] ?? "–";
     }
+
+    /**
+     * @param string $redirect_to
+     * @param string $action_name
+     * @param array  $post_ids
+     *
+     * @return string
+     */
+    public function handleBulkActions(string $redirect_to, string $action_name, array $post_ids): string
+    {
+        file_put_contents(
+            date('YmdHis') . "_handleBulkActions.json",
```

Comment:
Is dit nieuw? Worden deze bestanden automatisch weer weggehaald?

Reply:
moest weg ✅